deptofdefense/move.mil
# Add System Specs

## Checklist

I have…


## Summary of Changes

Rails 5.1 added [System Tests](http://guides.rubyonrails.org/testing.html#system-testing) to the framework and RSpec 3.7 added support for these kinds of tests. This pull request adds the necessary gems, configuration, etc. to support system tests and adds a system test for the Entitlements page.

It also adds a binstub for `rspec` so that invoking `bin/rspec` will run the test suite. You can also run individual specs by issuing commands like `bin/rspec spec/helpers/application_helper_spec.rb`.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout add-system-specs`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. run `bin/rspec`,
1. 🍏 🍏 🍏 ❗️
